src/olympia/constants/activity.py

```diff
@@ -1059,6 +1059,39 @@ class DENY_APPEAL_JOB(_LOG):
     reviewer_review_action = True
 
 
+class HELD_ACTION_ADMIN_USER_BANNED(_LOG):
+    id = 193
+    format = _('User {user} ban action held for further review.')
+    short = 'Held user ban'
+    admin_event = True
+
+
+class HELD_ACTION_DELETE_RATING(_LOG):
+    """Requires rating.id and add-on objects."""
+
+    id = 194
+    action_class = 'review'
+    format = _('Review {rating} for {addon} delete held for further review.')
+    reviewer_format = 'Held {user_responsible}s delete {rating} for {addon}'
+    admin_event = True
+
+
+class HELD_ACTION_COLLECTION_DELETED(_LOG):
+    id = 195
+    format = _('Collection {collection} deletion held for further review')
```

Comment:
I'm guessing this should have ended with a period like the other messages?

Reply:
We're not super consistent (`HELD_ACTION_FORCE_DISABLE` below, and the original `FORCE_DISABLE` are both missing `.` from the end of their format messages.).

Though as all of these are admin only events that shouldn't be exposed to end-users, they don't need to be localized anyway - they'll be a follow-up that drops them.

Reply:
> shouldn't be exposed to end-users, they don't need to be localized anyway - they'll be a follow-up that drops them.

These are being actively translated (see [latest commits](https://github.com/mozilla/addons-server/commits/master/)). If that's the intent, we should try not to expose them, vs dropping them later.